app/src/ui/app.tsx

```diff
@@ -1457,6 +1457,8 @@ export class App extends React.Component<IAppProps, IAppState> {
       title = __DARWIN__ ? 'No Repositories' : 'No repositories'
     }
 
+    const tooltip = repository ? repository.path : undefined
```

Comment:
I'd like for this to be only shown when the repository list isn't open. We have a check below for `isOpen` that you can use. Thoughts?

Reply:
Yes, showing a tooltip when dropdown is open is redundant and is unnecessary. I made the change. Thanks!